ppp_datamodel/utils.py

```diff
@@ -9,3 +9,33 @@ def predicate(node, childs):
         else:
             return any(childs.values())
     return tree.fold(predicate)
+
+def isincluded(tree1,tree2):
+    """
+        Return True if and only if tree1 is included in tree2.
+    """
+    if isinstance(tree1,Resource):
+        tree1=List([tree1])
+    if isinstance(tree2,Resource):
+        tree2=List([tree2])
+    if type(tree1) != type(tree2):
```

Comment:
What if some day I add a class `TypedMissing` that derived from `Missing`. This will become False, whereas it would return True if I was using a parent class.